extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/AwsXRayPropagator.java

```diff
@@ -123,18 +122,18 @@ public static AwsXRayPropagator getInstance() {
     Objects.requireNonNull(carrier, "carrier");
     Objects.requireNonNull(getter, "getter");
 
-    SpanContext spanContext = getSpanContextFromHeader(carrier, getter);
-    if (!spanContext.isValid()) {
+    Span span = extractSpan(carrier, getter);
```

Comment:
I'd like to separate out this changes into a follow-on PR, just to keep this one a little more tight in its scope.